ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-periodics.yaml

```diff
@@ -79,86 +79,6 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
-- agent: kubernetes
```

Comment:
cc @ReToCode

Reply:
Yes we need to create branches and CI for 1.13 as well. But we need to fix the linked issue first.